### PR TITLE
Tukan Theme Update for Sidebar

### PR DESCRIPTION
**Pull request by marjohn56**

### CODE REVIEW

**Comment by opnsenseuser** [comment]
@Franco 
5columns
5colums_source

5 columns would be worth considering. that would offer in my opinion now, since the screens in general tend to be wide. Also of the resolution. I have now taken the cicada theme as an example(screenshot). But of course it is possible for all themes. Worth a consideration? Pfsense can currently be adjusted to 6 columns. For monitoring purposes a must have. What do you think?

**Comment by mimugmail** [comment]
Beautiful screenshot! Gut gemacht Rene 👍

**Comment by fabianfrz** [comment]
looks good but keep in mind that IPv6 addresses are longer.

**Comment by fichtner** [comment]
@opnsenseuser yes, looks very nice. :) as users can select and save their column count it's ok to add more here. Do you want to do the change or should we?

File: misc/theme-tukan/src/opnsense/www/themes/tukan/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -1,8078 +1,9818 @@
-@charset "UTF-8"
```
Comment (fichtner) [inline misc/theme-tukan/src/opnsense/www/themes/tukan/assets/stylesheets/main.scss]: I feel bad for asking, but this also doesn't look like SCSS or does it?

**Comment by opnsenseuser** [comment]
would be great if you can do that. I'm on vacation for 2 weeks. best regards, rene

Freitag, 13. Juli 2018, 20:02 +0200 von Michael  <notifications@github.com>:

File: misc/theme-tukan/src/opnsense/www/themes/tukan/assets/stylesheets/main.scss (hunk 1 above)
Comment (fabianfrz) [reply misc/theme-tukan/src/opnsense/www/themes/tukan/assets/stylesheets/main.scss]: @fichtner you are right:
```
sassc main.scss 
Error: Invalid CSS after "...rder: 1px solid": expected expression (e.g. 1px, bold), was "#565656cc;"
        on line 7960 of main.scss
>>   border: 1px solid #565656cc;
   --------------------^
```

**Comment by opnsenseuser** [comment]
@martin. Please pr the Main.css file i sent to you a few days ago. Maybe that file works.
Rene



Gesendet von myMail für iOS


Freitag, 13. Juli 2018, 20:14 +0200 von Michael  <notifications@github.com>:

**Comment by marjohn56** [comment]
Corrected in main.scss and regenerated.

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**